CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ Changelog for the runtimes governed by the Polkadot Fellowship.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [Unreleased]
+
+### Changed
+
+- Apply patch for stable2409-6
```

Comment:
Why arent we going for 2412 directly https://github.com/polkadot-fellows/runtimes/pull/606 ?

Reply:
> Why arent we going for 2412 directly #606 ?

Sure, we could.  
Actually, I just wanted to prepare and clean up some things for Bridges, but aside from that, I proactively tried to clean this up in case we need additional hotfixes based on stable2409.  

However, I don't need this bump now, I've reverted Bridges changes here and picked the relevant Bridges changes into a separate PR: [#627](https://github.com/polkadot-fellows/runtimes/pull/627). So, this PR contains just the bump.

So, we can either merge this bump (just in case) or close it and wait for the [stable2412 PR](https://github.com/polkadot-fellows/runtimes/pull/606).

Reply:
let's get `2409-6` in so we make sure we get all fixes it comes with - if we manage to upgrade to `2412` in #606 soon that's great, but if that takes time we might want to do a patch release from existing `2409` for deploying various fixes/improvements

Reply:
Please use the default syntax for these changes.